### ISSUE TITLE
Network spoofer notvworking

### ISSUE THREAD

**Issue by Randzen** [w-shackleton/android-netspoof#99]
Network spoofer is not working at all. I have rooted galaxy grand prime and I have it all updated properly. What's wrong?

**Comment by w-shackleton** [comment]
Can you check you have root access with a root checker app?

**Comment by Randzen** [comment]
I do have root access i have checked but im using supersu and it says it
needs to update but never does??!

**Comment by w-shackleton** [comment]
I had this issue once with supersu - for me it was broken and didn't have correct root access. Perhaps try re-rooting?

**Comment by Randzen** [comment]
Yes I'm on kingroot now and only some of the networking spoofing works
